File: BPMOPMobile_VietNamAirLines/BPMOPMobile_VietNamAirLines/BPMOPMobile/Bean/ObjectSubmitDetailComment.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace BPMOPMobile.Bean
{
    public class ObjectSubmitDetailComment
    {
        public string ID { get; set; }
        public string DeviceName { get; set; } // Android: "App Android" - IOS: "App IOS"
        public string CodeName { get; set; } // giống DeviceName bỏ khoảng trắng: Android: "AppAndroid" - IOS: "AppIOS"
        public string AppName { get; set; } // Ex: "Digital Process"
        public string Platform { get; set; } // Tên hệ điều hành
        public string ResourceCategoryId { get; set; } // 8 WFItem - 16 Task
        public string ResourceUrl { get; set; } // Lấy trong BeanSettings - Task: TASK_URL - WFItem: WORKFLOW_URL
        public string ItemId { get; set; } // WFItem ID phiếu - Task ID Task
        public string Author { get; set; } // ID người tạo
        public string AuthorName { get; set; }

    }
}

File: VNABPMOP/app/src/main/java/com/vuthao/bpmop/base/model/custom/ObjectSubmitDetailComment.java
package com.vuthao.bpmop.base.model.custom;

public class ObjectSubmitDetailComment {
    private String ID;
    private String DeviceName; // Android: "App Android" - IOS: "App IOS"
    private String CodeName; // giống DeviceName bỏ khoảng trắng: Android: "AppAndroid" - IOS: "AppIOS"
    private String AppName; // Ex: "Digital Process"
    private String Platform; // Tên hệ điều hành
    private String ResourceCategoryId; // 8 WFItem - 16 Task
    private String ResourceUrl; // Lấy trong BeanSettings - Task: TASK_URL - WFItem: WORKFLOW_URL
    private String ItemId; // WFItem ID phiếu - Task ID Task
    private String Author; // ID người tạo
    private String AuthorName;

    public ObjectSubmitDetailComment() {
    }

    public String getID() {
        return ID;
    }

    public void setID(String ID) {
        this.ID = ID;
    }

    public String getDeviceName() {
        return DeviceName;
    }

    public void setDeviceName(String deviceName) {
        DeviceName = deviceName;
    }

    public String getCodeName() {
        return CodeName;
    }

    public void setCodeName(String codeName) {
        CodeName = codeName;
    }

    public String getAppName() {
        return AppName;
    }

    public void setAppName(String appName) {
        AppName = appName;
    }

    public String getPlatform() {
        return Platform;
    }

    public void setPlatform(String platform) {
        Platform = platform;
    }

    public String getResourceCategoryId() {
        return ResourceCategoryId;
    }

    public void setResourceCategoryId(String resourceCategoryId) {
        ResourceCategoryId = resourceCategoryId;
    }

    public String getResourceUrl() {
        return ResourceUrl;
    }

    public void setResourceUrl(String resourceUrl) {
        ResourceUrl = resourceUrl;
    }

    public String getItemId() {
        return ItemId;
    }

    public void setItemId(String itemId) {
        ItemId = itemId;
    }

    public String getAuthor() {
        return Author;
    }

    public void setAuthor(String author) {
        Author = author;
    }

    public String getAuthorName() {
        return AuthorName;
    }

    public void setAuthorName(String authorName) {
        AuthorName = authorName;
    }
}
